iree/turbine/kernel/wave/templates/extend_attention.py

```diff
@@ -344,6 +364,187 @@ def second_loop(
             res = tkw.cast(res, wave_output_dtype)
         tkw.write(res, c, mapping=mapping, elements_per_thread=STORE_ELEMS_PER_THREAD)
 
+    @tkw.wave(constraints)
```

Comment:
Do we need to create a separate kernel? Can we do what we did with the transposed-v where we kept the body the same?

Reply:
Done